nexus/src/db/model/mod.rs

```diff
@@ -211,6 +213,16 @@ macro_rules! impl_enum_type {
                 }
             }
         }
+
+        impl ToString for $model_type {
+            fn to_string(&self) -> String {
+                match self {
+                    $(
+                    $model_type::$enum_item => String::from_utf8($sql_value.to_vec()).unwrap(),
```

Comment:
There's two main parts making this unwrap safe:

1) the call to diesel::serialize::Output::write_all expects a `&[u8]`
1) the to_string fn calls `to_vec()` on $sql_value.

I've added `test_impl_enum_type_to_string` in 0afecd69 to make sure that of the macro_rule literals that do compile due to those two constraints, the to_string does not panic. This isn't a great test because it's not asserting compilation failure though.

Reply:
Note too that b083fe5266e4cc234935d4973e48f5102a88e137 changes to `from_utf8_lossy` and removes the unwrap.

Reply:
That seems better...but I'm still not sure what this is used for so I can't tell why it's safe to be lossy, either?